controls/srg_gpos.yml

```diff
@@ -27,5 +27,6 @@ controls:
             - var_account_disable_post_pw_expiration=35
             - var_auditd_action_mail_acct=root
             - var_auditd_space_left_percentage=25pc
+            - var_auditd_admin_space_left_percentage=5pc
```

Comment:
Why not to ad this to the relevant srg_gpos file in the subdirectory? It would be closer to the related rule. Or are you following some coding rule?